install_config/install/quick_install.adoc

```diff
@@ -166,10 +166,6 @@ an unattended installation, the installation fails.
 must be set to *true* for the configuration file to be considered valid.
 <10> If set to *true*, OpenShift services are run in a container on this host
 instead of installed using RPM packages.
-ifdef::openshift-enterprise[]
-Containerized installations are in Technology Preview in OpenShift Enterprise
-3.1.
-endif::[]
```

Comment:
Why was this removed? Looks like it was in your "fix conflicts" commit. This needs to be back in.